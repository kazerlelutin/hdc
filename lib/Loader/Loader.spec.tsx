import { render, cleanup } from '@testing-library/react';
import Loader from './Loader';

describe('<Loader />', () => {

    afterEach(cleanup);
    it('Check snapshot', () => {
        const { asFragment } = render(<Loader/>);
        expect(asFragment()).toMatchSnapshot();
    });
});
