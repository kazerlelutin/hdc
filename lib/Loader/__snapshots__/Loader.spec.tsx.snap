// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> Check snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
    data-testid="mug"
  >
    <div
      class="mugContainer"
    >
      <div
        class="mug"
      >
        <div
          class="coffee"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
